src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -1002,20 +1022,97 @@ class ShenandoahScanRemembered: public CHeapObj<mtGC> {
   void roots_do(OopIterateClosure* cl);
 };
 
+typedef struct ChunkOfRegion {
```

Comment:
Why `typedef`? Could this just be `struct ShenandoahRegionChunk`?

Reply:
Ok.  I've eliminated the typedef and replaced everywhere with struct ShenandoahRegionChunk.